### PR TITLE
Convert from task pane to content add-in

### PR DESCRIPTION
**Pull request by akrantz**
This project is based on https://github.com/OfficeDev/Office-Addin-TaskPane which represents a basic Office task-pane add-in.

This converts the project over to a content add-in.

### CODE REVIEW

**Review by arttarawork — APPROVED** [review]

**Review by TCourtneyOwen — APPROVED** [review]

File: src/content/powerpoint.ts [hunk 1]
```diff
@@ -26,4 +26,4 @@ export async function run() {
 
   await Office.context.document.setSelectedDataAsync(" ", options);
   await Office.context.document.setSelectedDataAsync("Hello World!", options);
-}
+}
```
Comment (akrantz) [inline src/content/powerpoint.ts:29]: need to add a newline here

File: test/src/powerpoint-test-content.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    await run();
+
+    // get selected text
+    const selectedText = await getSelectedText();
+
+    // send test results
+    testHelpers.addTestResult(testValues, "output-message", selectedText, "Hello World!");
+
+    await sendTestResults(testValues, port);
+}
```
Comment (akrantz) [inline test/src/powerpoint-test-content.ts:39]: need to add a newline here

File: test/src/test-content.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+import "./excel-test-content";
+import "./powerpoint-test-content";
```
Comment (akrantz) [inline test/src/test-content.ts:2]: need to add a newline here.

**Comment by TCourtneyOwen** [comment]
This is just to populate the repo right?  We shouldn't need this commit to be pristine

**Comment by arttarawork** [comment]
> This is just to populate the repo right? We shouldn't need this commit to be pristine

He's fixing up some minor merge conflicts

**Comment by akrantz** [comment]
The repository is populated with the taskpane/master branch. This is the pull request to convert from taskpane to content add-in.

I went through and made fixes and ran the tests.

**Pull request merged**